experiment/stop_experiment.py

```diff
@@ -26,9 +26,14 @@
 
 def stop_experiment(experiment_name, experiment_config_filename):
     """Stop the experiment specified by |experiment_config_filename|."""
+    experiment_config = yaml_utils.read(experiment_config_filename)
+
+    if experiment_config.get('local_experiment', False):
```

Comment:
can we use `experiment_utils.is_local_experiment()` here too?

Reply:
I prefer use the self-contained information within a function, rather than a global setting. wdyt?